xlml/apis/task.py

```diff
@@ -566,6 +568,7 @@ def post_process(
       return group
 
   def _get_job_manifest(self):
+    # pylint: disable=C0301
```

Comment:
what's this warning/error disabled? Could we add a comment? or to be more specific like `# pylint: disable=locally-disabled, multiple-statements, fixme, line-too-long`?

Reply:
Done.